code/modules/reagents/reagents/medical.dm

```diff
@@ -1183,3 +1183,87 @@
 			continue
 		limb_to_unfix.fracture()
 		break
+
+
+/datum/reagent/medicine/research
+	name = "Research precursor" //nothing with this subtype should be added to vendors
+	taste_description = "bitterness"
+	reagent_state = LIQUID
+	taste_description = "bitterness"
+
+
+/datum/reagent/medicine/research/quietus
+	name = "Quietus"
+	description = "This is a latent poison, designed to quickly and painlessly kill you in the event that you become unable to fight. Never washes out on it's own, must be purged."
+	color = "#19C832"
+	custom_metabolism = REAGENTS_METABOLISM * 0
+	taste_description = "Victory"
+
+/datum/reagent/medicine/research/quietus/on_mob_life(mob/living/L, metabolism)
+	switch(current_cycle)
+		if(1 to 59)
+			L.adjustStaminaLoss(1*effect_str)
+			if(prob(5))
+				to_chat(L, span_notice("You feel weakened by a chemical."))
+		if(60)
+			to_chat(L, span_notice("You feel the poison settle into your body."))
+		if(61 to INFINITY)
+			if(L.stat == UNCONSCIOUS)
+				L.adjustOxyLoss(25*effect_str)
+	return ..()
+
+/datum/reagent/medicine/research/somolent
+	name = "Somolent"
+	description = "This is a highly potent regenerative drug, designed to heal critically injured personnel. Only functions on unconscious or sleeping people."
+	color = "#19C832"
+	custom_metabolism = REAGENTS_METABOLISM * 2.5 //0.5u/ticc
+	taste_description = "naptime"
+
+/datum/reagent/medicine/research/somolent/on_mob_life(mob/living/L, metabolism)
+	switch(current_cycle)
```

Comment:
Switch statement for a single case?

Reply:
Because I want it to end after X... wait, can I just leave it as current_cycle? Self-taught here, and I've only seen switch(current_cycle) before.

Reply:
Just if(current_cycle < 51), yeah.

Reply:
So that would just be-
on mob life
*everything it does*
then *if current cycle* followed by nothing?